zgw/verzoek/build.gradle

```diff
@@ -43,6 +43,7 @@ dependencies {
     implementation project(":zgw:object-management")
     implementation project(":zgw:objecten-api")
     implementation "org.springframework.boot:spring-boot-starter"
+    implementation "org.springframework.boot:spring-boot-starter-data-jpa"
```

Comment:
`org.springframework:spring-tx` would suffice, but it doesn't bother me too much.